### ISSUE TITLE
Conv2DTranspose 

### ISSUE THREAD

**Issue by furkank75** [Dobiasd/frugally-deep#188]
Hi,
The repo is very nice! Top works, thanks for this.

My architecture includes a **Conv2DTranspose** Layer. It’s possible to add this Layer?

**Comment by Dobiasd** [comment]
Hi,

in many cases, it seems like `UpSampling2D` and `Conv2D` is a good alternative to using `Conv2DTranspose` (see here: https://github.com/Dobiasd/frugally-deep/blob/master/FAQ.md#why-are-conv2dtranspose-layers-not-
supported).

Could this be an option for you?

**Comment by Dobiasd** [comment]
Hi,

in many cases, it seems like `UpSampling2D` and `Conv2D` is a good alternative to using `Conv2DTranspose` (see here: https://github.com/Dobiasd/frugally-deep/blob/master/FAQ.md#why-are-conv2dtranspose-layers-not-supported).

Could this be an option for you?

**Comment by furkank75** [comment]
How can I do this with this codeline?
` Conv2DTranspose(8, (3, 3), strides=(2, 2), padding='same')`

**Comment by Dobiasd** [comment]
By using this (minimal working example):

```python3
import numpy as np
from keras.layers import Input, UpSampling2D, Conv2D
from keras.models import Model

inputs = Input(shape=(32, 32, 3))
x = inputs
x = Conv2D(8, (3, 3), padding='same')(UpSampling2D(2)(x))
model = Model(inputs=inputs, outputs=x)
```

instead of that:

```python3
import numpy as np
from keras.layers import Input, Conv2DTranspose
from keras.models import Model

inputs = Input(shape=(32, 32, 3))
x = inputs
x = Conv2DTranspose(8, (3, 3), strides=(2, 2), padding='same')(x)
model = Model(inputs=inputs, outputs=x)
```
